diff --git Package.swift Package.swift
index 56c3ee1..8e32086 100644
--- Package.swift
+++ Package.swift
@@ -45,7 +45,7 @@ let package = Package(
             name: "InjectionImpl",
             dependencies: ["InjectionImplC",
                 .product(name: "DLKitD", package: "DLKit"), // DEBUG_ONLY versions
-                .product(name: "SwiftRegexD", package: "SwiftRegex")]),
+                .product(name: "SwiftRegexD", package: "SwiftRegex5")]),
         // Boots up standalone injection on load for InjectionLite product
         .target(
             name: "InjectionBazel", dependencies: ["InjectionImpl",
